lib/ansiblelint/utils.py

```diff
@@ -533,6 +534,7 @@ def get_normalized_tasks(yaml, file):
     return res
 
 
+@lru_cache(maxsize=128)
```

Comment:
This is the same as above. Called only in `matchyaml` or `matchtasks`

Reply:
The only reason why I did not approve it is that I did not had time to test it myself. I want to fix 2.10 testing first and right after that this one.